openapi/types.go

```diff
@@ -42,6 +42,12 @@ type DataType interface {
 	Format() string
 }
 
+// Exampler is the interface implemented by types
```

Comment:
wdyt of "Exampler is the interface implemented by custom types that can parse example values." ?

Reply:
Sounds nice. Changed.